src/Compilers/Core/Portable/Operations/ISimpleAssignmentOperation.cs

```diff
@@ -16,6 +16,10 @@ namespace Microsoft.CodeAnalysis.Operations
     /// </remarks>
     public interface ISimpleAssignmentOperation : IAssignmentOperation
     {
+        /// <summary>
+        /// Is this a ref assignment
+        /// </summary>
+        bool IsRef { get; }
```

Comment:
>bool IsRef { get; } [](start = 8, length = 19)

Why not expose RefKind property instead?

Reply:
> Why not expose RefKind property instead?

Because the kind of the reference is irrelevant here. Even compiler doesn't capture the kind.